index2.html

```diff
@@ -0,0 +1,230 @@
+            <button class="resume">Get my resume</button>
+          </div>
+          <hr class="about-breaker">
+          <div class="skills">
+            <ul class="tools-list">
+                <ul class="listtopography">
+                 <li class="header-list" id="language">
+                  <img id="titles" src="img/Rectangle 55.png" alt="Languages">
+                  <h1>Language</h1>
+                 </li>
+                 <ul class="mini-list">
+                  <li class="detail">JavaScript</li>
+                  <li class="detail">Ruby</li>
+                  <li class="detail">HTML</li>
+                  <li class="detail">CSS</li>
+                 </ul>
+                </ul>
+                <ul class="listtopography">
+                  <li class="header-list" id="frameworks">
+                    <img id="titles" src="img/Rectangle 56.png" alt="Languages">
+                    <h1>Frameworks</h1>
+                  </li>
+                  <ul class="mini-list">
+                    <li class="detail">React.js</li>
+                    <li class="detail">Ruby on rails</li>
+                    <li class="detail">RSpec</li>
+                    <li class="detail">Capybara</li>
+                  </ul>
+                 </ul>
+                 <ul class="listtopography">
+                    <li class="header-list" id="skills">
+                      <img id="titles" src="img/Rectangle 57.png" alt="Languages">
+                      <h1>Skills</h1>
+                     </li>
+                     <ul class="mini-list">
+                      <li class="detail">Database Management</li>
+                      <li class="detail">Version Control</li>
+                      <li class="detail">CLI</li>
+                      <li class="detail">API Design</li>
+                      <li class="detail">Web Development</li>
+                     </ul>
+                  </ul>
+              </ul>
+          </div>
+       </section>
+       <section class="bottom-footer">
+           <div class="contact-section">
+              <p class="contact-text"> I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
+           </div>
+           <form action="https://formspree.io/html" method="post">
+            <fieldset> 
+            <label> 
+           <input id="Full-name" name="" type="text" required placeholder="Full name" max-length="30" />
+            </label>
+            <label>
+           <input id="email" name="" type="email" required placeholder="Email address" />
+          </label>
+          </fieldset>
+            <textarea name="" id="mess" placeholder="Write me something..." cols="50" rows="10" maxlength="500"></textarea>
```

Comment:
- [ ] Great job @Obimbo07 Your project looks good there are just a few style changes that the previous reviewer mentioned which haven't yet been implemented. :+1: 
- [ ] First of all, the previous reviewer mentioned that you style the input elements as seen in the Figma design but currently, you have a border around the textarea which wasn't present in the original design so kindly remove it :+1: 
- Your Design
![image](https://github.com/Obimbo07/Portfolio-mobileFirst/assets/102699958/d8cfc5dd-5829-466b-817e-118a23a420ea)

- The Figma Design
![image](https://github.com/Obimbo07/Portfolio-mobileFirst/assets/102699958/a7124c87-43ef-4c8a-9844-6d93380b267e)